### PR TITLE
Nisse/v0.7.5 albedo2d materialprops2d support in dgpv1 decoders

### PR DESCRIPTION
**Pull request by pd-nisse**

### CODE REVIEW

File: paralleldomain/common/dgp/v0/constants.py [hunk 1]
```diff
@@ -33,20 +33,20 @@
...
 
 class DirectoryName:
-    CALIBRATION: str = "calibration"
-    ONTOLOGY: str = "ontology"
-    RGB: str = "rgb"
-    POINT_CLOUD: str = "point_cloud"
     BOUNDING_BOX_2D: str = "bounding_box_2d"
     BOUNDING_BOX_3D: str = "bounding_box_3d"
-    SEMANTIC_SEGMENTATION_2D: str = "semantic_segmentation_2d"
+    CALIBRATION: str = "calibration"
```
Comment (pd-michaelstanley) [inline paralleldomain/common/dgp/v0/constants.py:38]: Should Albedo2D be in this DirectoryName somewhere?

File: paralleldomain/encoding/dgp/v1/scene.py [hunk 2]
```diff
@@ -1222,6 +1270,12 @@ def _prepare_output_directories(self) -> None:
                     (self._output_path / DirectoryName.KEY_LINE_3D / camera_name).mkdir(exist_ok=True, parents=True)
                 if AnnotationTypes.Polygons3D in self._annotation_types:
                     (self._output_path / DirectoryName.POLYGON_3D / camera_name).mkdir(exist_ok=True, parents=True)
+                if AnnotationTypes.Albedo2D in self._annotation_types:
```
Comment (pd-michaelstanley) [inline paralleldomain/encoding/dgp/v1/scene.py:1273]: Not your mistake, but while you're in there, L1247-1249 and L1253-1255 are duplicates.

File: paralleldomain/common/dgp/v0/constants.py (hunk 1 above)
Comment (pd-nisse) [reply paralleldomain/common/dgp/v0/constants.py:39]: Added to Encoder! thx

File: paralleldomain/encoding/dgp/v1/scene.py (hunk 2 above)
Comment (pd-nisse) [reply paralleldomain/encoding/dgp/v1/scene.py:1273]: nice catch thx

**Review by pd-michaelstanley — APPROVED** [review]

**Pull request merged**